packages/ui-components/src/components/Toolbar.tsx

```diff
@@ -22,21 +21,17 @@ const ToolbarWrapper = styled.div`
   }
 `;
 
-export const BaseToolbar = ({ children, maxWidth }) => (
+interface BaseToolbarProps {
+  maxWidth?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | false | undefined;
```

Comment:
can we use ContainerTypeMap.maxWidth? and maybe the default should be false instead of undefined?